LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
@hedss (better late than never I guess) I can't recall being part of this discussion tbh, so can't really enumerate ins and outs